### PR TITLE
Update .gitattributes file

### PR DESCRIPTION
**Pull request by ker0x**
All those files aren't required when fetching the package from composer!

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,6 +2,5 @@ composer.lock
 composer.phar
```
Comment (norkunas) [inline .gitignore:2]:
```suggestion
```
I think this can be removed

Comment (ker0x) [reply .gitignore]: Done!

**Comment by ker0x** [comment]
Tests failure are unrelated

**Review by norkunas — APPROVED** [review]

**Comment by norkunas** [comment]
> Tests failure are unrelated

I don't have enough permissions to merge when CI is red :D

**Comment by norkunas** [comment]
Please rebase :)

**Comment by ker0x** [comment]
Done!

**Review by norkunas — APPROVED** [review]

**Comment by norkunas** [comment]
Thank you @ker0x

**Pull request merged**